src/main/java/edu/cornell/kfs/gl/CuGeneralLedgerConstants.java

```diff
@@ -5,6 +5,7 @@
 public class CuGeneralLedgerConstants extends GeneralLedgerConstants {
 
 	public static final String ANNUAL_CLOSING_CHARTS_PARAM = "ANNUAL_CLOSING_CHARTS";
+	public static final String COLLECTOR_HEADER_RECORD_TYPE = "HD";
```

Comment:
This is the same value that the existing GL Collector parsing looks for when parsing. The base KFS code doesn't appear to have it defined in a variable (or at least in a publicly-available variable), which is why I added it to our GL constants.